code/collections.rb

```diff
@@ -1,15 +1,23 @@
 # collections
 
 # 1. Create a variable `point1` and assign it a hash representing a coordinate with keys `:x` and `:y` which have integer values
-
+point1 = {x:3, y:13}
 # 2. Print the y value
-
+puts point1[:y]
 # 3. Make an array, `points`, that includes `point1` and two more points (`points.length` should be 3)
-
+points = [point1, {x:4, y:14}, {x:5, y:15}]
 # 4. Use `.each` to print the y value of each point
-
+points.each do |numbers|
+  puts numbers[:y]
+end
 # 5. Use `.each_with_index` and string interpolation to print for each point "The point at index <index> has coordinates {x: <x-val>, y: <y-val>}"
-
+points.each_with_index do |numbers, index|
+  puts "The point at index #{index} has coordinates #{numbers}"
```

Comment:
👍 This works, but we wanted your `puts` string to look more like this:

```ruby
"The point at index #{index} has coordinates {x: #{numbers[:x]}, y: #{numbers[:y]}}"
```
can you spot the difference(s) with your version?
```ruby
"The point at index #{index} has coordinates #{numbers}"
```